src/transition/properties.js

```diff
@@ -52,7 +52,7 @@ function getTransitionProperties() {
 
   let vendorMap = {
     O: e => `o${e.toLowerCase()}`,
-    Moz: e => `moz${e}`,
+    Moz: e => e.toLowerCase(),
```

Comment:
yes, we changed the logic to better match what React does. which has a more generic "vendor prefix any event" utility. Doing more research tho you're right, that bites us here where there isn't actually a prefixed firefox transitionend.